RequestService/RequestService.Repo/Repository.cs

```diff
@@ -1788,7 +1790,7 @@ public async Task UpdateJobsToDoneFromInProgress()
             {
                 foreach (EntityFramework.Entities.Job job in jobs)
                 {
-                    await UpdateJobStatusDoneAsync(job.Id, -1, CancellationToken.None);
+                    await UpdateJobStatusDoneAsync(job.Id, -1, JobStatusChangeReasonCodes.AutoProgressingShifts, CancellationToken.None);
```

Comment:
Even here, it would perhaps be better to use a parameter.  With the current implementation, we are hard-coding an assumption that this method will only be called by an automated job.